src/routes/(blog-article)/how-to-setup-the-development-environment/+page.md

```diff
@@ -86,28 +83,39 @@ sudo apt-get install libsqlite3-dev
 
 </CodeBlock>
 
+  <!-- Colocarle la razon de su uso migracion de datos -->
+
 ## Set Up the Torrust Tracker
 
 At the time of writing, the Torrust Tracker requires:
 
-- rustc 1.72.0-nightly (839e9a6e1 2023-07-02)
+- rustc 1.77.1-nightly (839e9a6e1 2024-03-26)
```

Comment:
HI @ivan9989-pkj I think we can change that because the tracker works with previous versions. Maybe we could say:

"This tutorial has been tested with the following rust version:"